woocommerce/pinterest-for-woocommerce
# Feature/advertiser creation

Note: **This PR is branched off** #105. So testing of this PR and #105 can be combined. 

Implementation of [Automate advertiser creation flow](https://app.clickup.com/t/9r120q)

### Changes proposed in this pull request
Leverages the new capabilities of the Pinterest API  to handle advertiser creation and terms acceptance without interrupting the  flow or forcing the merchant to do it manually in Pinterest.

[Design reference](https://www.figma.com/file/ojzkxOhHQnIZx4QjsFH5Zy/Native-Pinterest-Integration?node-id=1636%3A35138)

#### Screenshots


### Detailed test instructions

1. Start the onboarding flow from scratch, use a new Pinterest account that doesn't have any advertisers already created.
2. Verify that on the "Setup tracking" step you are presented with the link that corresponds to the TOS of your country (see PDF doc above)
3. Verify that you cannot proceed unless you accept the terms. 
4. Verify that after you accept the terms and click continue, an advertiser and tag is created, and you can complete setup.
